File: VB/VB/WinSolution.Module.Win/ViewController1.vb
Imports Microsoft.VisualBasic
Imports System
Imports DevExpress.ExpressApp
Imports DevExpress.ExpressApp.Actions
Imports DevExpress.ExpressApp.Win.Templates

Namespace WinSolution.Module.Win
	Partial Public Class ViewController1
		Inherits ViewController
		Public Sub New()
			InitializeComponent()
			RegisterActions(components)
		End Sub
		Private Sub popupWindowShowAction1_CustomizeTemplate(ByVal sender As Object, ByVal e As CustomizeTemplateEventArgs) Handles popupWindowShowAction1.CustomizeTemplate
			CType(e.Template, PopupForm).Size = New System.Drawing.Size(200, 200)
		End Sub
		Private Sub popupWindowShowAction1_CustomizePopupWindowParams(ByVal sender As Object, ByVal e As CustomizePopupWindowParamsEventArgs) Handles popupWindowShowAction1.CustomizePopupWindowParams
			Dim os As ObjectSpace = Application.CreateObjectSpace()
			e.View = Application.CreateDetailView(os, os.CreateObject(Of DomainObject1)())
		End Sub
	End Class
End Namespace

File: CS/E523/WinSolution.Module.Win/ViewController1.cs
using DevExpress.ExpressApp;
using DevExpress.ExpressApp.Actions;
using DevExpress.ExpressApp.Win.Templates;

namespace WinSolution.Module.Win {
    public partial class ViewController1 : ViewController {
        public ViewController1() {
            InitializeComponent();
            RegisterActions(components);
        }
        private void popupWindowShowAction1_CustomizeTemplate(object sender, CustomizeTemplateEventArgs e) {
            ((PopupForm)e.Template).Size = new System.Drawing.Size(200, 200);
        }
        private void popupWindowShowAction1_CustomizePopupWindowParams(object sender, CustomizePopupWindowParamsEventArgs e) {
            ObjectSpace os = Application.CreateObjectSpace();
            e.View = Application.CreateDetailView(os, os.CreateObject<DomainObject1>());
        }
    }
}
